tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/uploadArtifactCli.ts

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+import * as fs from 'fs';
+import {
+    ArtifactBlobUploader,
+    ArtifactBlobUploaderContext,
+    ArtifactPipelineUploader,
+    ArtifactPipelineUploaderContext,
+    GenerateAndBuildOutput,
+    getGenerateAndBuildOutput,
+    logger,
+    requireJsonc,
+} from '@azure-tools/sdk-generation-lib';
+
+import {
+    uploadBlobInput,
+    UploadBlobInput,
+    uploadPipelineArtifactInput,
+    UploadPipelineArtifactInput,
+} from './cliSchema/uploadArtifactConfig';
+
+async function main() {
+    const args = parseArgs(process.argv);
+    const uploadTypt = args['uploadTypt'];
+
+    switch (uploadTypt) {
+        case 'blob':
+            uploadBlobInput.validate();
+            const config: UploadBlobInput = uploadBlobInput.getProperties();
+            if (!fs.existsSync(config.generateAndBuildOutputFile)) {
```

Comment:
Such kind of file existing check should place before send. That is before 'switch' block.

Reply:
done